cmd/config/set/set.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package set
+
+import (
+	"github.com/openshift/rosa/pkg/config"
+	"github.com/openshift/rosa/pkg/rosa"
+	"os"
+	"strconv"
+
+	"github.com/spf13/cobra"
+)
+
+var args struct {
+	debug bool
+}
+
+var Cmd = &cobra.Command{
+	Use:   "set [flags] VARIABLE VALUE",
+	Short: "Sets the variable's value",
+	Long:  "Sets the value of a config variable. See 'ocm config --help' for supported config variables.",
+	Args:  cobra.ExactArgs(2),
+	Run:   run,
+}
+
+func init() {
+	flags := Cmd.Flags()
+	flags.BoolVar(
+		&args.debug,
+		"debug",
+		false,
+		"Enable debug mode.",
+	)
+}
+
+func run(cmd *cobra.Command, argv []string) {
```

Comment:
@tirthct Same suggestion here to make use of a `ConfigService` that can `Save` the config given a `Config` struct e.g

```
type ConfigService interface {
  SaveConfig(config Config) error
}
```

Reply:
Same as above, we tried covering the method without the mock behavior